### ISSUE TITLE
Support for CylindricalMesh

### ISSUE THREAD

**Issue by RemDelaporteMathurin** [fusion-energy/openmc-dagmc-wrapper#102]
CylindricalMesh would be a very useful feature for fusion applications.

https://docs.openmc.org/en/stable/pythonapi/generated/openmc.CylindricalMesh.html#openmc.CylindricalMesh

**Comment by shimwell** [comment]
Yep we could get the bounding box to automatically define the height and radius. I don't have any methods of plotting the result but I'm sure we could make something
